src/amo/components/SearchForm/index.js

```diff
@@ -49,24 +50,34 @@ export class SearchFormBase extends React.Component<Props> {
   }
 
   render() {
-    const { className, i18n } = this.props;
+    const { className, i18n, apiLang, clientApp } = this.props;
 
     return (
-      <form
-        action={this.baseSearchURL()}
-        className={makeClassName('SearchForm', className)}
-        method="GET"
-        data-no-csrf
-        role="search"
-      >
-        <AutoSearchInput
-          inputName="q"
-          onSearch={this.onSearch}
-          onSuggestionSelected={this.onSuggestionSelected}
-          selectSuggestionText={i18n.gettext('Go to the add-on page')}
-          showInputLabel={false}
-        />
-      </form>
+      <React.Fragment>
+        <Helmet>
+          <link
+            title={`Firefox Add-ons (${apiLang || ''})`}
```

Comment:
We should use the `i18n` object instead of hard-coding the title here:

```js
i18n.sprintf(i18n.gettext('Firefox Add-ons (%(locale)s)'), { locale: apiLang })
```